### DISCUSSION TITLE
Add support for app monitoring platforms

### DISCUSSION THREAD

**Discussion by josecelano** [torrust/torrust-tracker#693] [Feature requests]
Like: https://sentry.io/

We are running the live demo: https://github.com/torrust/torrust-tracker/issues/601

Although we are using logs we do not get notification when there is a error. The only way to check for errors in production is to sign in to the server and parse log files.
 
For example:

```console
$ docker logs tracker | grep "[ERROR]"
$ docker logs index | grep "[ERROR]"
```

Currently, we don't have any errors in the 7-day logs. However, It would be nice to receive a notification when a new error is logged. In the past, I've used Sentry (for example). It would be easy to change the log setup to log errors to both the local log files and also send them to sentry. So that you don't need to find errors manually in the logs.

cc @torrust/torrustaceans

**Comment by mario-nt** [comment]
@josecelano I think it would be a nice addition.
